src/components/media-player/extensions/controls/controls.component.ts

```diff
@@ -45,8 +45,8 @@ export class MediaPlayerControlsExtensionComponent extends MediaPlayerBaseExtens
         }
     };
 
-    private _volume: number = 50;
-    private _previousVolume = 50;
+    private _volume: number = 100;
+    private _previousVolume = 100;
```

Comment:
I'm not clear why this was changed. Please update the "proposed changes" section in the PR if this was intentional.

Reply:
The default browser volume is 100 so by having 50 was causing an expression changed error. But as Donna mentioned it is good to put things like this in the Proposed Changes section

Reply:
Apologies, there was a ExpressionChangedAfterItHasBeenChecked error when running fixture.detectChanges() in the karma tests. 

These changes authored by @ashley-hunter are to prevent this error from occurring.

Reply:
No problem, could you also add repro details to the ticket to make sure that this gets proper QA?

Reply:
sure thing